Client/Frontend/Browser/Browser.swift

```diff
@@ -356,6 +360,27 @@ class Browser: NSObject {
             self.screenshotUUID = NSUUID()
         }
     }
+
+    @available(iOS 9, *)
+    func toggleDesktopSite(on: Bool) {
+        guard let webView = webView, let currentItem = webView.backForwardList.currentItem else {
+            return
+        }
+
+        if on {
+            let defaults = NSUserDefaults(suiteName: AppInfo.sharedContainerIdentifier())!
+            webView.customUserAgent = UserAgent.desktopUserAgent(defaults)
+        } else {
+            webView.customUserAgent = nil
+        }
```

Comment:
Then this could be reduced to: `webView.customUserAgent = on ? UserAgent.desktopUserAgent() : nil`.